docs/UserGuide.md

```diff
@@ -107,33 +107,42 @@ Examples:
 * `delete student m/CS2103 i/A1234567A`: Removes student with id A1234567A from module CS2103’s list of students
 * `delete student m/CS2100 i/A7654321A`: Removes student with id A7654321A from module CS2100’s list of students
 
-### Delete module from TAB: `delete module`
+### Edit a student's information: `edit student`
 
-Deletes the specified module from TAB
+Edits a student's information
 
-Format: `delete module m/module_name`
+Format: `edit student m/<module name> i/<student id> n/<edited student name> t/<edited student tele handle> e/<edited student email>`
 
 Examples:
-* `delete module m/CS2103`: Deletes module CS2103 from TAB
-* `delete module m/CS2100`: Deletes module CS2100 from TAB
+* `edit student m/CS2103 i/A0123456A n/John Doe t/@johndoe e/john.doe@u.nus.edu`: Edits student with id
+  A0123456A's information in module CS2103
+* `edit student m/CS2100 i/A0123457A n/Amy t/@amylee e/amy@u.nus.edu`: Edits student with id
+  A0123457A's information in module CS2100
 
-### Create a new student: `student add`
+### Find a student: `find`
 
-Create a new student
+Find the student in the module specified along with the student's information
 
-Format: `student add <student id>, <student name>, <student teleHandle>, <student email>`
+Format: `find m/<module name> i/<student id>`
 
 Examples:
-* `student add A0123456A, John Doe, @johndoe, john.doe@u.nus.edu`: add student John Doe
+* `find m/CS2103 i/A0123456A`: Finds and shows student with ID A0123456A from module CS2103
 
-### Get student information: `student get`
+### Delete module from TAB: `delete module`
 
-Get the student’s information (name, id, telehandle, email)
+Deletes the specified module from TAB
 
-Format: `student get <student id>`
+Format: `delete module m/module_name`
 
 Examples:
-* `student get A0123456A`: get information of the student with student id A0123456A
+* `delete module m/CS2103`: Deletes module CS2103 from TAB
+* `delete module m/CS2100`: Deletes module CS2100 from TAB
+
+### Go to homepage: `home`
+
+Returns to homepage
```

Comment:
Maybe can be more specific here? Instead of just saying homepage, can describe it briefly so that users know which one we are referring to.